lib/elixir/lib/kernel/cli.ex

```diff
@@ -183,54 +212,63 @@ defmodule Kernel.CLI do
   # Parse shared options
 
   defp parse_shared([opt | _t], _config) when opt in ["-v", "--version"] do
-    if function_exported?(IEx, :started?, 0) and IEx.started? do
-      IO.puts "IEx " <> System.build_info[:build]
+    if function_exported?(IEx, :started?, 0) and IEx.started?() do
+      IO.puts("IEx " <> System.build_info()[:build])
     else
-      IO.puts :erlang.system_info(:system_version)
-      IO.puts "Elixir " <> System.build_info[:build]
+      IO.puts(:erlang.system_info(:system_version))
+      IO.puts("Elixir " <> System.build_info()[:build])
     end
 
-    System.halt 0
+    System.halt(0)
   end
 
   defp parse_shared(["-pa", h | t], config) do
     paths = expand_code_path(h)
     Enum.each(paths, &:code.add_patha/1)
-    parse_shared t, %{config | pa: config.pa ++ paths}
+    parse_shared(t, %{config | pa: config.pa ++ paths})
   end
 
   defp parse_shared(["-pz", h | t], config) do
     paths = expand_code_path(h)
     Enum.each(paths, &:code.add_pathz/1)
-    parse_shared t, %{config | pz: config.pz ++ paths}
+    parse_shared(t, %{config | pz: config.pz ++ paths})
   end
 
   defp parse_shared(["--app", h | t], config) do
-    parse_shared t, %{config | commands: [{:app, h} | config.commands]}
+    parse_shared(t, %{config | commands: [{:app, h} | config.commands]})
   end
 
   defp parse_shared(["--no-halt" | t], config) do
-    parse_shared t, %{config | halt: false}
+    parse_shared(t, %{config | halt: false})
   end
 
   defp parse_shared(["-e", h | t], config) do
-    parse_shared t, %{config | commands: [{:eval, h} | config.commands]}
+    parse_shared(t, %{config | commands: [{:eval, h} | config.commands]})
   end
 
   defp parse_shared(["-r", h | t], config) do
-    parse_shared t, %{config | commands: [{:require, h} | config.commands]}
+    parse_shared(t, %{config | commands: [{:require, h} | config.commands]})
   end
 
   defp parse_shared(["-pr", h | t], config) do
-    parse_shared t, %{config | commands: [{:parallel_require, h} | config.commands]}
+    parse_shared(t, %{config | commands: [{:parallel_require, h} | config.commands]})
   end
 
-  defp parse_shared([erl, _ | t], config) when erl in ["--erl", "--sname", "--name", "--cookie", "--logger-otp-reports", "--logger-sasl-reports"] do
-    parse_shared t, config
+  @erl_shared_options [
+    "--erl",
+    "--sname",
+    "--name",
+    "--cookie",
+    "--logger-otp-reports",
+    "--logger-sasl-reports"
+  ]
+
+  defp parse_shared([erl, _ | t], config) when erl in @erl_shared_options do
+    parse_shared(t, config)
   end
```

Comment:
Formatter pulled this into a very large `when erl in [....]` block with lots of new lines, so I pulled it into `@erl_shared_options`.

I'm not _super_ familiar with this, but seemed like an appropriate variable name. Happy to change with better feedback

Reply:
It looks good.

Reply:
I like it too, good job!